.github/workflows/makefile.yml

```diff
@@ -0,0 +1,27 @@
+name: Makefile CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
nothings hppened on pull_request #3

Reply:
Problem was resolved ([Run#1](https://github.com/ivansoft/PortfolioAnalyser/actions/runs/1658616044)) by temporarily adding GH-3's head branch `pr-github-actions` to `push` trigger - see https://github.com/ivansoft/PortfolioAnalyser/pull/3/commits/8be6ac83129a79c48680ea8e0d3c5d13fd23e2bc:

> ```
>  push:
>    branches: [ master, pr-github-actions ]
> ```

Then branch `pr-github-actions` was removed from `push` event (see https://github.com/ivansoft/PortfolioAnalyser/pull/3/commits/8f44e594510eb2a2336f273a90f9ad4ae61a965f) but workflow continued to be called - [Run#20](https://github.com/ivansoft/PortfolioAnalyser/actions/runs/1669969986)